spec/space_spec.rb

```diff
@@ -8,4 +8,10 @@
   it 'returns a space' do
     expect(Space.create).to eq("Space")
   end
+
+  it 'knows its name' do
+    boardroom = Space.new("Makers")
```

Comment:
Find the choice of variable name, boardroom, a little confusing as the name of the space is 'Makers'.  Could the variable name just be space_1?  Sorry I know this is a bit pedantic - it just took a bit longer to understand it than maybe neccessary?